### PR TITLE
Move from Confluence Wiki to GitHub README

### PR DESCRIPTION
**Pull request by justinharringa**
Background: https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/

* Add CODEOWNERS for automatic PR assignment
* Add to Release Drafter:
  https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.adoc
* Archive CHANGELOG to pull into GitHub Releases
* Add new plugin info badges to README
* Pull in data from wiki using https://jenkins-wiki-exporter.g4v.dev/
* Change primary wiki to GitHub README in POM
* Improve main/resources/index.jelly

### CODE REVIEW

**Comment by justinharringa** [comment]
FYI @sortiz-sfdc @superfell 

Also, @superfell would you be interested in deleting your original repo? That would make this repo primary. The only PR there I noticed was adding proxy support which was handled in #2 (though it was handled by using the standard Jenkins configuration rather than by a custom field). As long as you don't make it private, I think that this would likely become the new primary [according to the GitHub doc](https://help.github.com/en/github/administering-a-repository/deleting-a-repository) and a [further explanation in the doc](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/what-happens-to-forks-when-a-repository-is-deleted-or-changes-visibility). My assumption would be that this could potentially win since it has more forks?

**Comment by superfell** [comment]
I deleted my repo, and the forked a copy of this one, but this repo doesn't appear to have become the new root.

**Comment by superfell** [comment]
Thanks for continuing to working on this.

File: README.md [hunk 1]
```diff
@@ -1,23 +1,58 @@
...
+The plugin is available via the main Jenkins plugin distribution, so go to the manage Jenkins page, manage plugins, and then the available tab and search for Chatter Notifier and pick
 the option to install it.
 
 # Configure
 
+## Credential Configuration
+Create new credentials of the kind "Username with Password"
+*  Remember to include your users API security token if needed as part of the password
+
+![Jenkins Credentials Example](images/jenkins-credentials.png)
```
Comment (sortiz-sfdc) [inline README.md]: This link doesn't appear to work in the rich-text display. Does the image exist?

Comment (justinharringa) [reply README.md]: I think the rich text view is looking for it in `master`.

Comment (justinharringa) [reply README.md]: I added it in this PR so that's why it's not showing up. 😀

**Comment by justinharringa** [comment]
@superfell I'll put in a ticket to GitHub support. Thanks for kicking this off! 😀

**Comment by justinharringa** [comment]
GitHub support made this the root of the network :boom:

**Pull request merged**